stacker/blueprints/redis.py

```diff
@@ -0,0 +1,101 @@
+from troposphere import Ref, ec2, Output, GetAtt
+from troposphere.elasticache import ReplicationGroup, SubnetGroup
+
+from .base import Blueprint
+
+SUBNET_GROUP = 'RedisSubnetGroup'
+SECURITY_GROUP = 'RedisSecurityGroup'
+REPLICATION_GROUP = 'RedisReplicationGroup'
+
+
+class RedisCluster(Blueprint):
+    PARAMETERS = {
+        'VpcId': {'type': 'AWS::EC2::VPC::Id', 'description': 'Vpc Id'},
+        'CacheClusterId': {
+            'type': 'String',
+            'description': 'The node group identifier',
+        },
+        'PrivateSubnets': {
```

Comment:
I would change this to Subnets - this lets you choose which subnets to put it in, rather than implicitly assuming private (which may be the safe case, but still).  This is a change I've been making in RDS as well.